### DISCUSSION TITLE
No module named 'prometheus_client'

### DISCUSSION THREAD

**Discussion by bbmh23** [pbrissaud/suivi-bourse#23] [Q&A] [answered]
Hello ! 
I've tried to use the last version of this app with docker-compose mode and I've two comments :

- there is a mistake in the docker-compose.yaml :  _paulbrissaud/suivi-bourse-app:v3.0.3_ isn't a not a tag which is recognized by the docker hub => the official tag for this version is _paulbrissaud/suivi-bourse-app:3.0.3_
- I'm not able to run the image _paulbrissaud/suivi-bourse-app:3.0.3_ in docker-compose mode and the logs shows the error : _File "/home/appuser/main.py", line 9, in <module> import prometheus_client ModuleNotFoundError: No module named 'prometheus_client'_

In advance, thanks for the help !

**Comment by pbrissaud** [comment, marked as answer]
Hi,
Sorry for the delay, I don't receive email notifications when discussion are opened. The bug is now fixed in version > 3.1.0

**Reply by bbmh23** [reply]
Hi ! 
No problem Paul ;-)  I've tried again and now it's OK.
Thanks
